### PR TITLE
fixes for https://gitcoincommunity.slack.com/archives/C8E45J5D0/p1525097521000711

### PR DESCRIPTION
**Pull request by owocki**
What

* fixes for https://gitcoincommunity.slack.com/archives/C8E45J5D0/p1525097521000711
* here is the issue that got screwed up in the first place
https://github.com/ethereum/solidity/issues/3965

How

* updates gitcoinbot comment to identify who has 'precedence' over an issue
* fixes some css issues with the 'start work' modal
* [Adds two checkboxes to the 'start work' modal](http://bits.owocki.com/31360K2N3u2C/Screen%20Shot%202018-04-30%20at%209.08.42%20AM.png)

### CODE REVIEW

**Comment by owocki** [comment]
@axic i will add a clause to https://gitcoin.co/help , but i am not optimistic that people are reading this content..

**Comment by owocki** [comment]
576a9de46ab8b8f8344d39ab84259702ea733339

File: app/dashboard/utils.py [hunk 1]
```diff
@@ -356,3 +356,21 @@ def build_profile_pairs(bounty):
             addr = f"https://etherscan.io/address/{fulfillment.fulfiller_address}"
             profile_handles.append((fulfillment.fulfiller_address, addr))
     return profile_handles
+
+
+def get_ordinal_repr(num):
```
Comment (mbeacom) [inline app/dashboard/utils.py]: Refactored this method a little bit.

File: app/assets/v2/css/bounty.css [hunk 2]
```diff
@@ -482,11 +482,21 @@ a.btn {
...
   width: auto !important;
 }
 
-.btn-interested {
+.btn-interested, .btn-interested:hover {
   border-radius:0.2em;
+  color: black;
+  border: black 1px solid;
+}
+.btn-interested:hover {
```
Comment (thelostone-mc) [inline app/assets/v2/css/bounty.css]: Ah empty line missed here

File: app/assets/v2/js/pages/bounty_details.js [hunk 3]
```diff
@@ -430,6 +430,18 @@ var show_interest_modal = function() {
         var has_question = event.target[0].value;
         var issue_message = event.target[2].value;
 
+        var agree_precedence = event.target[3].checked;
+        var agree_not_to_abandon = event.target[4].checked;
+
+        if (!agree_precedence) {
+          alert('You must agree to the precedence clause.');
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bounty_details.js]: Would it make sense to reuse our our custom alert like [here](https://github.com/gitcoinco/web/blob/master/app/assets/v2/js/pages/dashboard.js#L470)

`_alert({message: 'You must agree to the precedence clause.'}, 'error');`

File: app/assets/v2/js/pages/bounty_details.js [hunk 4]
```diff
@@ -430,6 +430,18 @@ var show_interest_modal = function() {
...
+          return false;
+        }
+        if (!agree_not_to_abandon) {
+          alert('You must agree not to keep the fulfiller updated on your progress.');
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bounty_details.js]: Same as [this](https://github.com/gitcoinco/web/pull/1024/files#r185189626)

**Comment by thelostone-mc** [comment]
@mbeacom I know this is merged  -> make sense adding what I'd commented on ?

**Pull request merged**